File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_advanced_-_exercise/fishing_boat.py
budget = int(input())
session = str(input())
count_fisher = int(input())

price = 0
discount = 0
discount_plus = 1
sum = 0

if count_fisher <= 6:
    discount = 0.10
    if session == "Spring":
        price = 3000
        if count_fisher % 2 ==0:
            discount_plus = 0.05

    elif session == "Summer":
        price = 4200
        if count_fisher % 2 ==0:
            discount_plus = 0.05

    elif session == "Autumn":
        price = 4200

    elif session == "Winter":
        price = 2600
        if count_fisher % 2 ==0:
            discount_plus = 0.05


elif 7 <= count_fisher <= 11:
    discount = 0.15
    if session == "Spring":
        price = 3000
        if count_fisher % 2 ==0:
            discount_plus = 0.05

    elif session == "Summer":
        price = 4200
        if count_fisher % 2 ==0:
            discount_plus = 0.05

    elif session == "Autumn":
        price = 4200

    elif session == "Winter":
        price = 2600
        if count_fisher % 2 ==0:
            discount_plus = 0.05

elif count_fisher >= 12:
    discount = 0.25
    if session == "Spring":
        price = 3000
        if count_fisher % 2 ==0:
            discount_plus = 0.05

    elif session == "Summer":
        price = 4200
        if count_fisher % 2 ==0:
            discount_plus = 0.05

    elif session == "Autumn":
        price = 4200

    elif session == "Winter":
        price = 2600
        if count_fisher % 2 ==0:
            discount_plus = 0.05

if discount_plus == 1:
    sum = price - (price * discount)

else:
    dis = price - (price * discount)
    sum = dis - (dis * discount_plus)

if budget >= sum:
    left = budget - sum
    print("Yes! You have " + "{:.2f}".format(left) + " leva left.")

elif budget < sum:
    need_more = sum - budget
    print("Not enough money! You need " + "{:.2f}".format(need_more) + " leva.")

File: JS/JavaScript/JS-Basics/Conditional StatementsAdvancedExercise/FishingBoat.js
function boat(inputs) {

    let budget = Number(inputs[0]);
    let session = String(inputs[1]);
    let people = Number(inputs[2]);

    let amount;
    let discount;

    if (people <= 6) {
        discount = 0.10;
    }

    else if (people > 7 && people <= 11) {
        discount = 0.15;
    }

    else if (people > 12) {
        discount = 0.25;
    };

    if (session == "Spring") {
        let boat = 3000;

        amount  = boat - boat * discount;
    }
    else if (session == "Summer" ) {
        let boat = 4200;

        amount  = boat - boat * discount;
    }

    else if (session == "Autumn") {
        let boat = 4200;

        amount  = boat - boat * discount;
    }

    else if (session == "Winter") {
        let boat = 2600;

        amount  = boat - boat * discount;
    };

    if (people % 2 == 0 && session != "Autumn") {
        amount -= amount * 0.05
    }

    if (amount <= budget) {
        console.log(`Yes! You have ${(budget - amount).toFixed(2)} leva left.`);
    }

    else if (amount > budget) {
        console.log(`Not enough money! You need ${(amount - budget).toFixed(2)} leva.`)
    }
}

boat(["3000",
"Summer",
"11"])
